analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -667,7 +667,11 @@ def add_arguments_to_parser(parser):
                                default=argparse.SUPPRESS,
                                help="Force the running analyzers to use "
                                     "almost every checker available. The "
-                                    "checker groups 'alpha.', 'debug.' and "
+                                    "checker groups 'alpha.', 'debug.'"
+                                    "'osx.', 'abseil-', 'android-',"
+                                    "'darwin-', 'objc-', 'cppcoreguidelines-',"
+                                    "'fuchsia.', 'fuchsia-', 'hicpp-', 'llvm-',"
+                                    "'llvmlibc-', 'google-', 'zircon-'"
```

Comment:
```suggestion
                                    "checker groups 'alpha.', 'debug.', "
                                    "'osx.', 'abseil-', 'android-', "
                                    "'darwin-', 'objc-', 'cppcoreguidelines-', "
                                    "'fuchsia.', 'fuchsia-', 'hicpp-', 'llvm-', "
                                    "'llvmlibc-', 'google-', 'zircon-'"
```
Also fix it in the documentation.